custom/icds_reports/models/aggregate.py

```diff
@@ -137,7 +137,11 @@ class CcsRecordMonthly(models.Model):
     valid_in_month = models.SmallIntegerField(blank=True, null=True)
     mobile_number = models.TextField(blank=True, null=True)
     preg_order = models.SmallIntegerField(blank=True, null=True)
-    home_visit_date = models.DateField(blank=True, null=True)
+    home_visit_date = models.DateField(
```

Comment:
was this added in a separate migration or view change?

I think if you run `./manage.py makemigrations icds_reports` now, it will create a new migration udpating this model (even though this is an unmanaged model)

Reply:
it wasnt, but i ran that command and it didnt add those to the migration (which surprised me because i was expecting it to). oddly it did at it for two of the ls ones

Reply:
interesting. I thought it did that even for help_text updates